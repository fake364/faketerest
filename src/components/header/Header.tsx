import React from 'react';

type Props = {};

const Header: React.FC<Props> = ({ children }) => {
	return <header>{children}</header>;
};

export default Header;
