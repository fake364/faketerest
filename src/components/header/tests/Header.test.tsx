import React from 'react';
import { shallow } from 'enzyme';
import Header from '../Header';

describe('Header', () => {
  it('should render header with content', () => {
    const wrapper = shallow(<Header>Text</Header>);
    expect(wrapper).toMatchSnapshot();
  });
});
