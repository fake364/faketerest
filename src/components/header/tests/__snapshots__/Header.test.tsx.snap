// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render header with content 1`] = `
<header
  className="bg-white"
>
  Text
</header>
`;
